Exercise01/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Exercise01.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise01
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var list = (new CreateData()).GetList();
+            list.ForEach(Console.WriteLine);
+            Console.WriteLine();
+
+            var newList = list.Where(x => x.Score >= 8).OrderBy(x => x.Birthday).ToList();

```

Comment:
bad naming `newList`. It should be `goodStudents`

Reply:
Done